### PR TITLE
DSERV-121 TopLD adaptor changes to new schema

### PR DESCRIPTION
**Pull request by pedrohr**

### CODE REVIEW

File: data/adapters/topld_adapter.py [hunk 1]
```diff
@@ -1,40 +1,74 @@
...
+
+# Position,rsID,MAF,REF,ALT,Uniq_ID,VEP_ensembl_Gene_Name,VEP_ensembl_Consequence,CADD_phred,fathmm_XF_coding_or_noncoding,FANTOM5_enhancer_expressed_tissue_cell
+# 5031031,rs1441313282,0.010486891385767793,C,T,5031031:C:T,FP565260.3|FP565260.3|FP565260.3|FP565260.3|FP565260.3,"intron_variant|intron_variant|intron_variant|intron_variant,NMD_transcript_variant|intron_variant",2.135,.,.
 
 
 class TopLD(Adapter):
-    DATASET = 'topld'
+    DATASET = 'topld_linkage_disequilibrium'
+
+    def __init__(self, filepath_prefix=None, chr='all', ancestry='SAS'):
```
Comment (mingjiecn) [inline data/adapters/topld_adapter.py:21]: chr can't be 'all' if this is used to build variant id?

File: data/adapters/topld_adapter.py [hunk 2]
```diff
@@ -1,40 +1,74 @@
...
-                    _source = row[0]
-                    _target = row[1]
-                    label = 'topld'
+                    _source = self.ids[row[0]]['variant_id']
+                    _target = self.ids[row[1]]['variant_id']
+                    label = 'topld_linkage_disequilibrium'
                     _props = {
                         'chr': self.chr,
                         'negated': row[6] == '+',
                         'variant_1_base_pair': ':'.join(row[2].split(':')[1:3]),
```
Comment (mingjiecn) [inline data/adapters/topld_adapter.py:68]: variant_1_base_pair is the same with variant_2_base_pair?

**Review by mingjiecn — CHANGES_REQUESTED** [review]
have a couple questions in my comment.

File: data/adapters/topld_adapter.py (hunk 2 above)
Comment (pedrohr) [reply data/adapters/topld_adapter.py:68]: ooo good catch

File: data/adapters/topld_adapter.py (hunk 1 above)
Comment (pedrohr) [reply data/adapters/topld_adapter.py:21]: I will move `chr` to a regular parameter and remove the default value

**Review by mingjiecn — APPROVED** [review]

**Pull request merged**